torch/_export/converter.py

```diff
@@ -363,6 +401,7 @@ def convert_aten_div(self, node: torch._C.Node):
 
         self.convert_aten_op(node)
 
+    @_register("prim::If")
```

Comment:
I feel like there's some redundancy in _register("xxx") and its corresponding method and the registry. 

A simpler approach might just be:
```python
def convert_node(self, node):
  def _normalize(node_kind):
    ...
    # replace `::` with `_`

  return getattr(self, _normalize("prim::xxx"))(node)
```

Supporting conversion more node_kind would be  just adding a method to the converter class. Might need a seperate NodeConverter to isolate the namespace to avoid method collision.

Reply:
```python
def convert_tuple_construct(self):
  self.construct_iterable()

def convert_list_construct(self):
  self.construct_iterable()
```